Client/Assets/Images.xcassets/tabs_private.imageset/Contents.json

```diff
@@ -0,0 +1,13 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x",
+      "filename" : "tabs_private.png"
```

Comment:
Remove this from this PR (along with tabs_private.png)? Not related, and not used anywhere.